var/spack/repos/builtin/packages/scorep/package.py

```diff
@@ -93,8 +95,10 @@ def url_for_version(self, version):
     # SCOREP 8
     depends_on("binutils", type="link", when="@8:")
     depends_on("otf2@3:", when="@8:")
-    depends_on("cubew@4.8:", when="@8:")
-    depends_on("cubelib@4.8:", when="@8:")
+    depends_on("cubew@4.8.2:", when="@8.3:")
+    depends_on("cubelib@4.8.2:", when="@8.3:")
+    depends_on("cubew@4.8:", when="@8:8.1")
+    depends_on("cubelib@4.8:", when="@8:8.1")
```

Comment:
Since the 8.2 tarball still seems to be online, we should probably use `@8:8.2` here.